File: Python Programming/problems on pattern printing/problem38.py
'''
36. Accept number of rows and number of columns from user and display below pattern. 
   Input :  iRow = 4  iCol = 3  
  Output : 
				5 4 3 2 1   
				5 4 3 2 1   
				5 4 3 2 1 
	
	Author : Milind Nalawade
	
	Date : 01/07/2021
	
	Language : Python 3.9.0
	
'''
#====
#Logic
#====
def Print(row,col):
	no=col
	for i in range(0,row):
		for j in range(0,col):
			print(no," ",end=" ")
			no-=1
		no=col	
		print()	

#=============
#Entry point function
#=============

def main():
	row=int(input("Enter number of rows :"))
	col=int(input("Enter number of columns : "))
	Print(row,col)

#=====
#starter
#=====
if __name__=="__main__":
	main()

File: C++ Programming/problem on pattern printing/problem38.cpp
/*
38. Accept number of rows and number of columns from user and display below pattern. 
   Input :  iRow = 4  iCol = 3  
  Output :
				5 4 3 2 1   
				5 4 3 2 1   
				5 4 3 2 1 
	
	Author : Milind Nalawade
	
	Date : 01/07/2021

	Language : C++ programming

*/
using namespace std;
#include<iostream>

//Logic
void Print(int row,int column)
{
	int i=0,j=0,no=column;
	
	for(i=0;i<row;i++)
	{
		for(j=0;j<column;j++)
		{
			cout<<" "<<no<<" ";
			no--;
		}
		no=column;
		cout<<"\n";
	}
}

//Entry point function

int main()
{
	int row=0,column=0;
	cout<<"Enter row : ";
	cin>>row;
	cout<<"Enter column : ";
	cin>>column;
	Print(row,column);
	return 0;
}	